lib/elixir/lib/list.ex

```diff
@@ -577,6 +577,51 @@ defmodule List do
   end
 
   @doc """
+  If `enum` starts with `prefix`, return true. Otherwise, return false.
+  Note that `prefix` must be a list; only `enum` is able to be an arbitrary enumerable.
```

Comment:
I think we need this line  a new paragraph, otherwise it will be part of the summary IIRC how ExDoc works.